packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2031,6 +1925,130 @@ export class ContainerRuntime
 		await this.garbageCollector.initializeBaseState();
 	}
 
+	private async initializeSummarizer(loader: ILoader): Promise<void> {
+		const { maxOpsSinceLastSummary = 0, initialSummarizerDelayMs = 0 } = isSummariesDisabled(
+			this.summaryConfiguration,
+		)
+			? {}
+			: {
+					...this.summaryConfiguration,
+					initialSummarizerDelayMs:
+						// back-compat: initialSummarizerDelayMs was moved from ISummaryRuntimeOptions
+						//   to ISummaryConfiguration in 0.60.
+						this.runtimeOptions.summaryOptions.initialSummarizerDelayMs ??
+						this.summaryConfiguration.initialSummarizerDelayMs,
+				};
+		const module = await import(
```

Comment:
I am not sure I understand which part is delay loaded here (like the description says). Could you please add some more information in the description to help understand how delay loading will work. Like, the conditios where they will be loaded vs not loaded and when, etc.

Reply:
done